app/api/schema/event_statistics.py

```diff
@@ -51,7 +61,19 @@ def sessions_rejected_count(self, obj):
         return Session.query.filter_by(event_id=obj.id, state='rejected').count()
 
     def speakers_count(self, obj):
-        return Speaker.query.filter_by(event_id=obj.id).count()
+        accepted = SessionsSpeakersLink.query.filter_by(event_id=obj.id, session_state='accepted').count()
+        confirmed = SessionsSpeakersLink.query.filter_by(event_id=obj.id, session_state='confirmed').count()
+        pending = SessionsSpeakersLink.query.filter_by(event_id=obj.id, session_state='pending').count()
```

Comment:
Why not using the functions written?

Reply:
@iamareebjamal my bad, I thought I removed the functions, I will do that now. They are just redundant

Reply:
Created a single function which takes a parameter which varies as accepted, pending, etc